src/modules/fw_att_control/fw_att_control_main.cpp

```diff
@@ -800,7 +799,7 @@ FixedwingAttitudeControl::task_main()
 			}
 
 			/* if we are in rotary wing mode, do nothing */
-			if (_vehicle_status.is_rotary_wing) {
+			if (_vehicle_status.is_rotary_wing && !_vehicle_status.is_vtol) {
```

Comment:
Not sure if this will have any other side-effects yet. Without this fw attitude data isn't published until the VTOL switches to FW mode which results in the VTOL code reading the last known state when blending FW and MC during a transition resulting in offsets or reversed travel in control surface movements.